tests/appsec/waf/blocked.v1.html

```diff
@@ -0,0 +1,99 @@
+<!-- Sorry, you’ve been blocked -->
```

Comment:
Why is this not updated ? it still has the weird backtick

Reply:
See `blocked.v1.html`. I tagged in the repo `v1` as the versions that were in https://github.com/DataDog/asm-blocked-response-template just before the change, and `v2` the ones that just changed. And an additional `v0` because we had something prior in Java.